dotnet/iwebapi/.template.config/template.json

```diff
@@ -41,11 +41,15 @@
       "choices": [
         {
           "choice": "net5.0",
-          "description": "Target net5.0"
+          "description": "Target .NET 5"
```

Comment:
I don't think we should support .net 5 at all going forward. Supporting multiple frameworks with the templates would potentially require a lot of maintenance. If they want an older framework, they should use an older template version.